app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/InlineEdit/CancelButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Ok for this one... and the confirm... these do a lotta exact same stuff... i know breaking everything up is awesome... butttttt if these could be combined and added in the file that they'll be consumed?